SQL_SELECT_HW

h6bi opened the pull request.

h6bi committed:
added week 2 HW

KseniyaLazarchik reviewed (commented):
Hi @h6bi 

Well done 
Some issues 

Send everything in one file and indicate your name/surname in the file name.
Plz specify types of joins.

**Part 1**

1. Fine but don't forget to use UPPER/LOWER 

2. Payment should be processed through rental and inventory tables;
Group by store_id column, not by concatenated address; 
IFNULL is used to handle NULL.

3. Good, I like that you pick all the actors with the same rating instead of just using LIMIT 5

4. You also can use Crosstab or SUM + CASE but I like way you handling NULLs 

**Part 2**

1. Store_id from inventory table should be used 

2. Again not a mistake in this task, but remember that just LIMIT 5 will cut off films with same rating

**Part 3**

You only did V1, but that's okay. Good job

h6bi committed:
WEEK 2 HW correction nr 1

Hi!

Made the requested corrections, and uploaded it in one file.

I have one question about your review:
In part 1.2 what do you mean by "IFNULL is used to handle NULL.". As far is I know in Postgresql COALESCE is used instead of IFNULL.

Thanks! :)

h6bi commented:
Hi @KseniyaLazarchik !

I have commited the requested corrections.

I have one question about your review:
In part 1.2 what do you mean by "IFNULL is used to handle NULL.".? As far is I know in Postgresql COALESCE is used instead of IFNULL.

Thanks! :)

KseniyaLazarchik commented:
> Hi @KseniyaLazarchik !
> 
> I have commited the requested corrections.
> 
> I have one question about your review: In part 1.2 what do you mean by "IFNULL is used to handle NULL.".? As far is I know in Postgresql COALESCE is used instead of IFNULL.
> 
> Thanks! :)

Yeah, SQL dialects just got mixed up in my head, happens when you work across different DB)

KseniyaLazarchik reviewed (approved):
Good job

You can merge
